src/EditorFeatures/Core.Cocoa/NavigationCommandHandlers/FindBaseSymbolsCommandHandler.cs

```diff
@@ -84,15 +84,15 @@ protected override bool TryExecuteCommand(int caretPosition, Document document,
                             }
 
                             var definitionItem = overriddenSymbol.ToNonClassifiedDefinitionItem(document.Project.Solution, true);
-                            await context.OnDefinitionFoundAsync(definitionItem, cancellationToken).ConfigureAwait(false);
+                            await context.OnDefinitionFoundAsync(solution, definitionItem, cancellationToken).ConfigureAwait(false);
```

Comment:
instead of passing along a DefinitionItem (or SourceReferenceItem) which holds onto the Solution, we now pass along a lighterweight object with just the DocId and we also pass along the SOlution as a method-param for clients that need it.  but that way they can grab what they need and hold onto the DefItem without rooting the solution.